data/uhc_pack/functions/create_scoreboards.mcfunction

```diff
@@ -18,6 +18,7 @@ scoreboard objectives add uhcGlow dummy
 scoreboard objectives add uhcEnabled dummy
 scoreboard objectives add uhcPotion dummy
 scoreboard objectives add uhcDiaChest dummy
+scoreboard objectives add uhcHealth dummy
```

Comment:
`uhcHealth` is already used the name of a scoreboard and is used to display health in the tab list (see line 33-34). This will need to use a different scoreboard name instead. Possibly `uhcStHealth`